honeybadgermpc/batch_reconstruction.py

```diff
@@ -81,15 +81,30 @@ def attempt_reconstruct_batch(data, field, n, t, point):
     n_chunks = bs[0]
     assert all(b == n_chunks for b in bs)
     recons = []
+
+    if point.use_fft:
+        # Precompute data that can be reused for multiple chunks in a batch
+
+        poly = polynomials_over(field)
+        zs = [i for i in range(len(data)) if data[i] is not None]
+
+        assert len(zs) >= t + 1
+        zs = zs[:(t + 1)]
+        as_, ais_ = fnt_decode_step1(poly, zs, point.omega2, point.order)
+        precomputed_data = (zs, as_, ais_)
+    else:
+        precomputed_data = None
+
     for i in range(n_chunks):
         chunk = [d[i] if d is not None else None for d in data]
         try:
-            p, failures = attempt_reconstruct(chunk, field, n, t, point)
+            p, failures = attempt_reconstruct(chunk, field, n, t, point,
```

Comment:
code style: I think there are certain guidelines around how you make the split to the next line..
I follow this
```
attempt_reconstruct(
chunk, field, n, t, point, precomputed_data)
```

The other option is to move each parameter to a separate line. What you have used is inconsistent with the rest of the code, can you please change this?